ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/MutableBlockchain.java

```diff
@@ -58,6 +71,8 @@ void unsafeImportBlock(
 
   Difficulty calculateTotalDifficulty(final BlockHeader blockHeader);
 
+  Difficulty calculateTotalDifficultyForSyncing(BlockHeader blockHeader);
```

Comment:
Don't think this needs to be exposed in the interface

Reply:
DONE